src/main/java/com/coveo/spillway/storage/LimitUsageStorage.java

```diff
@@ -120,6 +120,28 @@
   Map<LimitKey, Integer> debugCurrentLimitCounters();
 
   /**
+   * @param resource The resource for which you want to get the current limit counts
+   * @return A Map of all the enforced limits for a resource and their current count
+   *
+   */
+  Map<LimitKey, Integer> getCurrentLimitCounters(String resource);
```

Comment:
Wouldn't it had been better to have only calls taking in all required stuff and returning an `Integer`? Returning the data as a Map looks a bit raw to me...